book/04-git-server/sections/protocols.asc

```diff
@@ -82,7 +94,7 @@ Instead of having to set up different URLs for these things, you can now use a s
 If you try to push and the repository requires authentication (which it normally should), the server can prompt for a username and password.
 The same goes for read access.
 
-In fact, for services like GitHub, the URL you use to view the repository online (for example, ``https://github.com/schacon/simplegit[]'') is the same URL you can use to clone and, if you have access, push over.
+In fact, for services like GitHub, the URL you use to view the repository online (for example, ``https://github.com/schacon/simplegit'') is the same URL you can use to clone and, if you have access, push over.
```

Comment:
🤔 Why is it undesirable for this link to be clickable?

Reply:
Whoops, sorry, I had no idea that's what it meant, I thought it was a typo. Still learning asciidoc. will revert.

Reply:
Yeah, it's the square brackets that do it, and it still works inside quotes. Here's what the version on `master` looks like [rendered in HTML](https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols#_smart_http):

![image](https://user-images.githubusercontent.com/39902/30786621-983bd454-a12d-11e7-87ab-cd763167ec2e.png)